access/acceptance/permissions_test.go

```diff
@@ -91,3 +92,78 @@ func TestAccDatabricksPermissionsResourceFullLifecycle(t *testing.T) {
 		},
 	})
 }
+
+func TestAccDatabricksJobPermissionsResourceFullLifecycle(t *testing.T) {
+	randomName := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	acceptance.AccTest(t, resource.TestCase{
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+				resource "databricks_job" "this" {
+					name = "First %[1]s"
+					timeout_seconds = 3600
+					max_retries = 1
+					max_concurrent_runs = 1
+
+					new_cluster  {
+						num_workers   = 1
+						spark_version = "6.6.x-scala2.11"
+						node_type_id  = "Standard_DS3_v2"
```

Comment:
replace with existing function to pick node type based on cloud used. see other tests for example.

#346 should eventually simplify it